templates/charter/wg-charter-template.md

```diff
@@ -0,0 +1,40 @@
+# WG #WGNAME Charter
```

Comment:
I'm not convinced that working groups should require a charter, especially one with a sponsor from the steering committee. I think as long as the special interest groups are approved, then they could be able to autonomously spawn working groups